modules/ROOT/pages/_partials/cookbook-dw/format-dates-ex01/transform.dwl

```diff
@@ -0,0 +1,8 @@
+%dw 2.0
+output application/json
+---
+[
+  formattedDate: |2020-10-01T23:57:59| as String {format: "uuuu-MM-dd"},
+  formattedTime: |2020-10-01T23:57:59| as String {format: "KK:mm:ss a"},
+  formattedDateTime: |2020-10-01T23:57:59| as String {format: "KK:mm:ss a, MMMM dd, uuuu"}
+]
```

Comment:
I'd replace the array of objects with just an object, it'll make the output more compact:

```
{
  formattedDate: |2020-10-01T23:57:59| as String {format: "uuuu-MM-dd"},
  formattedTime: |2020-10-01T23:57:59| as String {format: "KK:mm:ss a"},
  formattedDateTime: |2020-10-01T23:57:59| as String {format: "KK:mm:ss a, MMMM dd, uuuu"}
}
```